toolkit/tools/internal/retry/retry_test.go

```diff
@@ -0,0 +1,83 @@
+package retry
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Delay vs. retry count for mult = 1.0, delay = .1 seconds
+var expectedValuesMillis1 = []int{0, 100, 200, 300, 400, 500}
+
+// Delay vs. retry count for exp = 2.0, delay = 0.1 seconds
+var expectedValuesMillis2 = []int{0, 100, 400, 900, 1600, 2500}
+
+func TestCalcDelay1(t *testing.T) {
+	mult := 1.0
+	sleep := time.Millisecond * 100
+	for i, expected := range expectedValuesMillis1 {
+		expected := time.Duration(expected) * time.Millisecond
+		actual := calcDelay(i, sleep, mult)
+		assert.Equal(t, expected, actual)
+	}
```

Comment:
Could we use full words to make it clearer what the variables mean? Also, let's be consistent with the naming:
```suggestion
	backoffExponentBase := 1.0
	sleep := time.Millisecond * 100
	for i, expected := range expectedValuesMillis1 {
		expected := time.Duration(expected) * time.Millisecond
		actual := calcDelay(i, sleep, backoffExponentBase)
		assert.Equal(t, expected, actual)
	}
```